packages/plugin-ext/package.json

```diff
@@ -7,10 +7,12 @@
   "dependencies": {
     "@theia/core": "^0.3.11",
     "@theia/filesystem": "^0.3.11",
-    "@theia/workspace": "^0.3.11",
-    "@theia/plugin": "^0.3.11",
     "@theia/monaco": "^0.3.11",
+    "@theia/plugin": "^0.3.11",
+    "@theia/workspace": "^0.3.11",
+    "@types/lodash.clonedeep": "^4.5.3",
```

Comment:
should @types be moved to dev dependency ?

Reply:
thanks, I'll fix it